lib/api_repo.js

```diff
@@ -1018,11 +1021,14 @@ ApiRepo.prototype._makeProtocFunc = function _makeProtocFunc(opts, language) {
           pluginOption = '--plugin=';
         } else {
           args.push('--' + language + '_out=' + outDir);
-          if (language === 'python') {
-            // required per https://github.com/grpc/grpc/issues/7857
-            args.push('--grpc_python_out=' + outDir);
-          } else {
-            args.push('--grpc_out=' + outDir);
+          if (!opts.buildCommonProtos ||
+              _.includes(COMMON_GRPC_PROTOS, protoPath)) {
+            if (language === 'python') {
+              // required per https://github.com/grpc/grpc/issues/7857
+              args.push('--grpc_python_out=' + outDir);
```

Comment:
I thought we decided not to do this in order to support GAE?

My understanding was that this required a more complex change to generate both gRPC and gRPC-less modules.